userguide/subscription/includes/catalog-introduction.adoc

```diff
@@ -1,78 +1,76 @@
 
 * **Plan Cancellation Timing** - Should the customer get a pro-ration credit when the subscription is cancelled before its next renewal date?
 * **Plan Change Timing** - Should a customer downgrading to a lower (paid) plan be active immediately or should the system wait for the next renewal date?
 
 
-The catalog is also used by the invoicing system to make sure each each `Customer` is billed at the right time and for the right amount. Information about **how** the customer is billed and **when** the customer is billed is also defined in the catalog using alignments and their associated rules.
+The catalog is also used by the invoicing system to make sure each `Customer` is billed at the right time and for the right amount. Information about **how** the customer is billed and **when** the customer is billed is also defined in the catalog using alignments and their associated rules.
 
-Catalog configuration allows to define if different subscriptions attached to a customer should be invoiced at the same time, or at the opposite if they should be billed separately on their own schedule (or a mix of both where some are grouped together on the same invoice and others are invoiced independently). In the simplest case, where all subscriptions are grouped together and there is a `BCD` defined (`Billing Period` is a multiple of the month), the customer gets one invoice per `Billing Period` on that (`BCD`) day. Note that grouping subscriptions together may by default lead to some leading pro-ration when the subscription is started on a different date than the `BCD` (as the system needs to realign invoicing on that particular date). This behavior is configured in the catalog through the http://docs.killbill.io/latest/userguide_subscription.html#_billing_alignment_rules[Billing Alignment Rules].
+Catalog configuration allows the Kill Bill Administrator to define whether all subscriptions attached to a customer should be invoiced at the same time, or if they should be billed separately on their own schedule. There could also be a mix of both where some are grouped together on the same invoice and others are invoiced independently. In the simplest case, where all subscriptions are grouped together and there is a Bill Cycle Day (BCD) defined (`Billing Period` is a multiple of the month), the customer gets one invoice per `Billing Period` on that day. Note that grouping subscriptions together may by default lead to some proration when the subscription is started on a different date than the BCD, as the system needs to realign invoicing on that particular date. This behavior is configured in the catalog through the http://docs.killbill.io/latest/userguide_subscription.html#_billing_alignment_rules[Billing Alignment Rules].
```

Comment:
> In the simplest case, where all subscriptions are grouped together and there is a Bill Cycle Day (BCD) defined (`Billing Period` is a multiple of the month), the customer gets one invoice per `Billing Period` on that day.

I find this a tad confusing as I believe it mixes two (closely related) topics.

On one hand, we support various `Billing Period` (defined [here](https://github.com/killbill/killbill-api/blob/master/src/main/java/org/killbill/billing/catalog/api/BillingPeriod.java)), which can (but not necessarily) be month-aligned (monthly, bimestrial, etc.). Most of this paragraph applies (i.e. you can group weekly subscriptions together).

On the other hand, we have a concept of a `bill cycle day`, which is the preferred day in the month when a subscription (or group of subscriptions) needs to be invoiced. This day can be global for the account or defined on a per subscription (or subscription bundle) basis.

@sbrossie Should we be more explicit here and explain both topics separately?

Reply:
@pierre @jdmooney I think we should indeed explain this. However this section, `How is the Catalog Used Inside Kill Bill?` is part of the Introduction, and so perhaps not the best place to go so deep. Maybe we should even simplify this section, and instead add more details in our `Billing Alignment Rules` section?

Reply:
@pierre @sbrossie I agree that it would be best to keep introductory parts simple, with more detail later on. How about removing the discussion of the BCD here, and moving it to the `Billing Alignment Rules` section as @sbrossie suggested?